### PR TITLE
chore: migrate to generator-videojs-plugin 7.6.3

### PR DESCRIPTION
**Pull request by brandonocasey**
## Description
Migrate to the newest version of `generator-videojs-plugin` so that this project is up to date with current conventions.

## Things of note
* Builds scripts were changed to be actual scripts, rather than needing to be required
* Moved test data creation to `test/dist` rather than just `test`

## TODO (in another pull request)
* Move test data creation into a rollup plugin
* Move `utils/manifest` to `test/manifest` as its weird to have test assets in `utils`
* Remove custom-assertions, as they modify qunit assert, which is really strange. Just make them helpers.
* Get tests running on ie11 and edge
* Update sinon

### CODE REVIEW

File: test/playback.test.js [hunk 1]
```diff
@@ -237,7 +237,7 @@ QUnit.test('DASH sidx', function(assert) {
 
     // Firefox sometimes won't loop if seeking directly to the duration, or to too close
     // to the duration (e.g., 10ms from duration). 100ms seems to work.
-    player.currentTime(player.duration() - 0.1);
+    player.currentTime(player.duration() - 0.5);
```
Comment (gkatsev) [inline test/playback.test.js]: are we having problems with 0.1?

File: scripts/create-test-segments.js [hunk 2]
```diff
@@ -72,15 +83,7 @@ module.exports = {
       console.log('files in segments dir were changed rebuilding segments data');
       this.build();
     });
-  },
-
-  clean() {
```
Comment (gkatsev) [inline scripts/create-test-segments.js]: why was clean removed? Should --clean still be an option above?

File: package.json [hunk 3]
```diff
@@ -53,71 +47,73 @@
...
   },
   "files": [
-    "dist/"
+    "CONTRIBUTING.md",
+    "dist/",
+    "docs/",
+    "index.html",
+    "scripts/",
+    "src/",
+    "test/"
```
Comment (gkatsev) [inline package.json]: We have a lot of fixtures in test/, probably not needed for published release

File: package.json [hunk 4]
```diff
@@ -12,37 +12,31 @@
...
-    "clean:dist": "rimraf dist dist-test",
-    "clean:test": "node scripts/clean-tests.js",
-    "postclean": "mkdirp dist",
-    "docs": "run-p docs:*",
-    "docs:api": "jsdoc src -r -d docs/api",
-    "docs:toc": "doctoc README.md",
-    "docs:nomnoml:build": "node -e \"var b=require('./scripts/nomnoml.js'); b.build();\"",
+    "clean": "shx rm -rf ./dist ./test/dist",
+    "postclean": "shx mkdir -p ./dist ./test/dist",
+    "build:test-manifests": "node ./scripts/create-test-manifests.js",
```
Comment (gkatsev) [inline package.json]: I don't think these two need to be run on `npm run build`, just on test should be enough

Comment (gkatsev) [reply package.json]: I would also prefer not to build the test dist file on just regular build but that sounds like it should happen in a separate PR

File: test/playback.test.js (hunk 1 above)
Comment (brandonocasey) [reply test/playback.test.js]: I was seing failures in firefox with this number every few runs, due to the increase in playbackRate.

File: scripts/create-test-segments.js (hunk 2 above)
Comment (brandonocasey) [reply scripts/create-test-segments.js]: These are now created in test/dist, which we cleanup

File: package.json (hunk 4 above)
Comment (brandonocasey) [reply package.json]: Well since we build test assets on build we have to do them on build for now.

File: scripts/create-test-segments.js (hunk 2 above)
Comment (gkatsev) [reply scripts/create-test-segments.js]: So, we don't need lines 37-40?

**Pull request merged**